### PR TITLE
Add delivery stats to endpoint

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddDeliveryStatsToEndpoint < ActiveRecord::Migration[5.2]
+  def change
+    add_column :endpoints, :last_delivery_status, :string
```
Comment (adamruzicka) [inline db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb]: Do we expect more than 3 states? If not, we could probably get away with making this column boolean+NULL

Comment (ShimShtein) [reply db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb]: Somehow string status feels better. Maybe later we will add more information about the delivery there.
If you insist, I'll make it boolean.

**Comment by ShimShtein** [comment]
Fixed rubocop

File: db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb (hunk 1 above)
Comment (adamruzicka) [reply db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb]: No problem with that, just wanted to know what was the intent

**Pull request merged**